crispy_bulma/templates/bulma/widgets/file_upload_input.html

```diff
@@ -1,17 +1,33 @@
+{% if widget.is_initial %}
+  {{ widget.initial_text }}:
+  <a href="{{ widget.value.url }}">
+    {{ widget.value }}
+  </a>
+  {% if not widget.required %}
+  <input type="checkbox" name="{{ widget.checkbox_name }}" id="{{ widget.checkbox_id }}"{% if widget.attrs.disabled %} disabled{% endif %}>
+  <label for="{{ widget.checkbox_id }}">
+    {{ widget.clear_checkbox_label }}
+  </label>
+  {% endif %}
+{% endif %}
 <div class="field">
-  <div class="file is-primary has-name">
+  <div class="file is-primary has-name is-small">
```

Comment:
Hello,

This PR is very good.
I just have one question: why `is-small` here ?

Reply:
I've removed it, because the default rendering should work without modifications for most people. Currently customizations can be fairly easy made by overwriting the `file_upload_input.html` template. Later we can create a custom bulma FileField.

Also I've also removed the `class="field"` wrapper, because this code is already wrapped. The official bulma docs do not have an example how  a form  with a FileField should look like, so I'm only 90% sure it is correct. The styled form looks good.